flank-scripts/src/main/kotlin/flank/scripts/integration/IssueList.kt

```diff
@@ -0,0 +1,21 @@
+package flank.scripts.integration
+
+import com.github.kittinunf.result.getOrElse
+import com.github.kittinunf.result.onError
+import flank.scripts.github.getGitHubIssueList
+
+suspend fun checkForOpenedITIssues(token: String) = getGitHubIssueList(
+    githubToken = token,
+    parameters = listOf(
+        "creator" to "github-actions[bot]",
+        "state" to "open",
+        "labels" to "IT_Failed"
+    )
+)
+    .onError { println(it.message) }
+    .getOrElse { emptyList() }
+    .getOrNull(0)
```

Comment:
do we need both `getOrElse` and `getOrNull` ?

Reply:
Hm...`getOrElse` (fuel's method) prevents ugly stack trace from being printed when there is no response (list) returned. `getOrNull` prevents from out of boundary exception (`get(0)`/`[0]`) or no such element exception (`first()`).
I don't know if there is fuel's method reflects 'get me first element from a returned list or null if there is no list at all or is empty'.
Or I can't find it 🤷‍♂️

Reply:
oh sorry I missed this `0 `🤦 
so would not it better to use `firstOrNull` ?

Reply:
I like `firstOrNull` thanks!